docs/source/quickstart/99_FAQ.md

```diff
@@ -77,22 +83,28 @@ failed: [hypervisor] (item=osp_trunk) => {
 }
 ```
 
-Cause: there's currently a conflict between the `default` network created by libvirt during its first startup, and the `osp_trunk` network
+~~~{admonition} Cause
+:class: error
+There's currently a conflict between the `default` network created by libvirt during its first startup, and the `osp_trunk` network
 we create within the CI Framework: they are both consuming the same range, `192.168.122.0/24`. This is mostly due to hard-coded values in the
 product, not the Framework. We're working on getting rid of them.
+~~~
 
-Solution: connect to the hypervisor, and issue this command:
+~~~{admonition} Solution
+:class: tip
```

Comment:
same this is rendered as is :class:

Reply:
see comment above at https://github.com/openstack-k8s-operators/ci-framework/pull/824/files#r1400073810

marking this one resolved